spec/services/taxonomy/taxon_fetcher_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require 'gds_api/test_helpers/publishing_api_v2'
+
+RSpec.describe Taxonomy::TaxonFetcher do
+  include GdsApi::TestHelpers::PublishingApiV2
+  describe 'taxons' do
+    let(:linkables) do
+      [
+        {"title" => "foo", "base_path" => "/foo", "content_id" => SecureRandom.uuid},
+        {"title" => "bar", "base_path" => "/bar", "content_id" => SecureRandom.uuid},
+        {"title" => "aha", "base_path" => "/aha", "content_id" => SecureRandom.uuid},
+      ]
+    end
+    
+    before do
+      publishing_api_has_linkables(linkables, document_type: 'taxon')
+    end
+
+    it 'retrieves content items from publishing api and orders by title' do
+      result = described_class.new.taxons
```

Comment:
This project mostly follows the style where `it` blocks contain all information needed to read the tests, without any "mystery guests" (from https://robots.thoughtbot.com/lets-not).

What would you think of inlining the `let` and `before` block?

```ruby
it 'retrieves content items from publishing api and orders by title' do
  linkables = [
    {"title" => "foo", "base_path" => "/foo", "content_id" => SecureRandom.uuid},
    {"title" => "bar", "base_path" => "/bar", "content_id" => SecureRandom.uuid},
    {"title" => "aha", "base_path" => "/aha", "content_id" => SecureRandom.uuid},
  ]
  
  publishing_api_has_linkables(linkables, document_type: 'taxon')

  result = described_class.new.taxons

  expect(result.first["title"]).to eq("aha")
  expect(result.last["title"]).to eq("foo")
end
```